modules/holiday-snow.php

```diff
@@ -31,16 +31,44 @@ public function blog_field_html() {
 	}
 
 	public function holiday_snow_option_updated() {
+		
+		/**
+		 * Fires when the holiday snow option is updated.
+		 * 
+		 * @since 2.0.3
+		 */
 		do_action( 'jetpack_holiday_snow_option_updated' );
 	}
 }
 
 function jetpack_holiday_snow_script() {
+	
+	/**
+	 * Filter the holiday snow.
+	 * 
+	 * There's no actual "chance" for snow, it's either on or off.
```

Comment:
I would disagree. defining this filter to return `true` doesn't mean that there will be snow falling on your site. You still have to enable the snow option in Settings > Reading in your dashboard.